### ISSUE TITLE
Some datasets' explorer_url do not end with slash

### ISSUE THREAD

**Issue by ebezzi** [chanzuckerberg/single-cell-data-portal#1526]
Currently, when you open an URL in explorer, it only works if there is a trailing slash. Some explorer_urls in prod do not have it and this might cause problems. We should:

1. Wait for https://github.com/chanzuckerberg/single-cell-explorer/pull/112 - this might solve the issue
2. QA this in dev by removing an explorer_url's trailing slash and testing if it works
3. If it doesn't work, find all the explorer_urls in prod that do not end with slash
4. Either create a cxg_admin script to migrate them or do it manually

**Comment by danieljhegeman** [comment]
I think `1.` will solve the problem. Will confirm once we merge which should be soon; waiting on final reviews.

**Comment by tihuan** [comment]
@danieljhegeman @leslieypark @ebezzi  should we disable strict forward slash globally, or is there a reason to keep it strict?

https://stackoverflow.com/a/44038627

Thank you!

**Comment by metakuni** [comment]
Until the fix is implemented/deployed in Explorer, we can run the script mentioned by Leslie above. i.e.:

**Comment by metakuni** [comment]
Moving this back into the product backlog for now as discussed with Arathi.

**Comment by maniarathi** [comment]
Closing as there are no other links in the _database_ that do not contain a trailing slash. Therefore all links from the Portal are valid links to Explorer.

The trailing slash issue is still a minor issue but it's a separate issue.
